config/galaxy.ini.sample

```diff
@@ -1230,20 +1220,23 @@ use_interactive = True
 # variables).
 #expose_potentially_sensitive_job_metrics = False
 
-# Data manager configuration options
+# Enable the API for sample tracking
+#enable_legacy_sample_tracking_api = False
+
+# -- Data manager configuration options
+
 # Allow non-admin users to view available Data Manager options.
 #enable_data_manager_user_view = False
+
 # File where Data Managers are configured (.sample used if default does not
 # exist).
 #data_manager_config_file = config/data_manager_conf.xml
+
 # File where Tool Shed based Data Managers are configured.
 #shed_data_manager_config_file = config/shed_data_manager_conf.xml
```

Comment:
Add blank line here too?

Reply:
@nsoranzo Thanks, missed that one in my reformatting!